docs/UserGuide.md

```diff
@@ -179,7 +179,7 @@ Removes an existing tag from an existing person in the address book.
 * `untag INDEX [t/TAG]…​`
```

Comment:
Should we include a note saying that at least 1 tag must be provided? `untag INDEX [t/TAG]` suggests that the `t/TAG` is optional

Reply:
Updated it to `untag INDEX t/TAG [t/MORE_TAGS]…​`